environment.yml

```diff
@@ -20,6 +20,7 @@ dependencies:
   - distributed
   - scikit-image
   - cython
+  - click
```

Comment:
how quickly is the ``click`` API evolving? would it make sense to stick to a specific version?

Reply:
Not certain, but I think the authors are very aware of the impact of API changes; so I think it's low risk. In general I would stick to "latest release" until things break, but that IMHO.

Reply:
no, OK, that's fine